hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java

```diff
@@ -66,6 +68,12 @@
     @Rule
     public final OverridePropertyRule overrideTaskSecondsRule = set(PROP_TASK_PERIOD_SECONDS, "1");
 
+    @Rule
+    public final OverridePropertyRule overrideCleanupPercentage = set(PROP_CLEANUP_PERCENTAGE, "100");
+
+    @Rule
+    public final OverridePropertyRule overrideCleanupOperationCount = set(PROP_CLEANUP_OPERATION_COUNT, "1000");
+
```

Comment:
While you're at it, you can use `smallInstanceConfig()` in this and the other test as the partition/event  count

Reply:
done.